RecyclerView/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">RecyclerView</string>
+
+    <string name="states_lover">I love %1s</string>
```

Comment:
This is good!
but,when single argument, no need to `1`.
you can rewrite `%s`.

https://developer.android.com/guide/topics/resources/string-resource.html